package.json

```diff
@@ -1,13 +1,12 @@
 {
-  "name": "openzeppelin-eth",
-  "version": "2.2.0",
+  "name": "@openzeppelin/contracts-ethereum-package",
+  "version": "2.2.1",
```

Comment:
Shouldn't this be a release candidate, if the dependencies we're using are also release candidates?